src/test/java/net/snowflake/ingest/streaming/internal/SnowflakeStreamingIngestChannelTest.java

```diff
@@ -49,9 +49,9 @@ public void testChannelFactoryNullFields() {
     String dbName = "DATABASE";
     String schemaName = "SCHEMA";
     String tableName = "TABLE";
-    Long channelSequencer = 0L;
-    Long rowSequencer = 0L;
-    SnowflakeStreamingIngestClientInternal<VectorSchemaRoot> client =
+    long channelSequencer = 0L;
+    long rowSequencer = 0L;
```

Comment:
why we're changing this?

Reply:
Do you mean `Long -> long`? My motivation was that there is no need to box non-null primitives.
I can roll it back if needed